csrf_test.go

```diff
@@ -41,6 +43,49 @@ func TestProtect(t *testing.T) {
 	}
 }
 
+// TestCookieOptions is a test to make sure the middleware correctly sets cookie options
+func TestCookieOptions(t *testing.T) {
+	s := http.NewServeMux()
+	s.HandleFunc("/", testHandler)
+
+	r, err := http.NewRequest("GET", "/", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	rr := httptest.NewRecorder()
+	expectedExpires := time.Now().Add(time.Duration(173) * time.Second)
+	p := Protect(testKey, CookieName("nameoverride"), Secure(false), HttpOnly(false), Path("/pathoverride"), Domain("domainoverride"), MaxAge(173))(s)
+	p.ServeHTTP(rr, r)
+
+	if rr.Header().Get("Set-Cookie") == "" {
+		t.Fatalf("cookie not set: got %q", rr.Header().Get("Set-Cookie"))
+	}
+
+	cookie := rr.Header().Get("Set-Cookie")
+	if strings.Contains(cookie, "HttpOnly") {
+		t.Fatalf("cookie does not respect HttpOnly option: got %v", cookie)
```

Comment:
Can you update this and subsequent tests to show `got %q want %q` to make it easier to debug & align with the other tests.